course_feature_tags.json

```diff
@@ -0,0 +1,282 @@
+[
```

Comment:
I wonder if it might be better to have this in a `featuretags.py` file, with the features, subfeatures, and tags represented by other constants.  This should reduce the possibility of a typing error if we ever need to add/modify the content.  But if that's unlikely to happen it may be overkill.

```python
ATTR_FEATURE = "ocw_feature"
ATTR_SUBFEATURE = "ocw_subfeature"
ATTR_TAG = "course_feature_tag"

FEATURE_FACULTY_INTRO = "AV faculty introductions"
FEATURE_AV_LECTURES = "AV lectures"
.....etc

SUBFEATURE_VIDEO = "Video"
SUBFEATURE_AUDIO = "Audio"
SUBFEATURE_NONE = ""
...etc

TAG_COURSE_INTRO = "Course Introduction"
TAG_LECTURE_VIDEOS = "Lecture Videos"
...etc

TAG_MAPPING = [
  {
    ATTR_FEATURE: FEATURE_FACULTY_INTRO,
    ATTR_SUBFEATURE: SUBFEATURE_NONE,
    ATTR_TAG: TAG_COURSE_INTRO
  },
  {
    ATTR_FEATURE: FEATURE_FACULTY_INTRO,
    ATTR_SUBFEATURE: SUBFEATURE_VIDEO,
    ATTR_TAG: TAG_COURSE_INTRO
  },
  {
    ATTR_FEATURE: FEATURE_AV_LECTURES,
    ATTR_SUBFEATURE: SUBFEATURE_VIDEO,
    ATTR_TAG: TAG_LECTURE_VIDEOS
  },.....etc]
```